test/test_um2netcdf.py

```diff
@@ -1303,7 +1303,20 @@ def test_enum_action_choices(enum_parser, test_enum):
     enum_action = [act for act in enum_parser._actions if act.dest == "enum"][0]
     assert enum_action.choices == tuple(c.value for c in test_enum)
 
-def test_enum_action_no_enum_type(enum_parser, test_enum):
+def test_enum_action_choices_set(test_enum):
+    """
+    Test that the EnumAction sets the correct choices.
+    """
```

Comment:
Just a tiny suggestion, would something like "Test that the EnumAction produces an error if `choices` keyword supplied" help with clarity here – it's not a big thing though and can easily be gathered from looking at the code.

Reply:
Right, after the additions it does both, so I will change to:
"Test that the EnumAction sets the correct choices and that it produces an error if 'choices' keyword is supplied."

Well spotted!

Reply:
You're right, it was copied from the one above 😅 
Well spotted!

Fixed in [091c905e6e97001c55e45842e38b2379f604cf24](https://github.com/ACCESS-NRI/um2nc-standalone/pull/160/commits/091c905e6e97001c55e45842e38b2379f604cf24)

Reply:
Awesome! This PR looks all good to me!